README.md

```diff
@@ -1,4 +1,4 @@
-# Gluten: Plugin to Double SparkSQL's Performance
+# Gluten: A Middle Layer for Offloading JVM-based SQL Engines' Execution to Native Engines
 [![OpenSSF Best Practices](https://www.bestpractices.dev/projects/8452/badge)](https://www.bestpractices.dev/projects/8452)
 *<b>This project is still under active development now, and doesn't have a stable release. Welcome to evaluate it.</b>*
```

Comment:
please also help to insert one blank before this line to address the issue of sentence continuity for OpenSSF badge and the next sentence.

Reply:
Fixed, thanks!